rfc/rfc002-authentication-token.md

```diff
@@ -129,26 +129,13 @@ EN:PractitionerLogin:v2 Undersigned gives permission to Nuts foundation to make
 
 The time layout used is constructed as: `Monday, 2 January 2006 15:04:05` The name of the service provider must match the name that has been registered for this service provider. The name of the care organisation must match the name within the proof that has been published by the service provider. The signature must be made by cryptographic means which can be connected to the user.
 
-## 6. Authentication Token Container
+## 6. Verifiable Presentation
 
-The Authentication Token MUST be signed by the user. See section 7 for more detail on supported means. In order to support several signing means, the signed token MUST be encapsulated by a Authentication Token Container.
+The Authentication Token MUST be signed by the user. See section 7 for more detail on supported means. In order to support several signing means, the signed token MUST be encapsulated by a Container. As a container specification we use the [W3C Verifiable Presentation specification 1.0](https://www.w3.org/TR/vc-data-model/).
 
-The container MUST be formatted as a JSON object with the following members.
+It is recommended to reuse existing specifications for Verifiable Presentations (VP). If no existing specification exists, the type of the VP MUST be prepended with `Nuts`. This indicates that the VP is specified within this RFC.
```

Comment:
> MUST be prepended with `Nuts`

Does casing matter here? Do we prefer 'nuts' rather than 'Nuts'?

> This indicates that the VP is specified within this RFC

I'd say SHOULD or MUST be specified by this RFC (not just indicating)

Reply:
The casing is in line with the VC spec (It uses upper camelcase everywhere)

Reply:
Changed to SHOULD